app/src/main/java/org/gem/indo/dooit/views/custom/NonSwipableViewPager.java

```diff
@@ -0,0 +1,57 @@
+package org.gem.indo.dooit.views.custom;
```

Comment:
Apparently smooth scroller is a hack for lower API version. It appears that without the custom scroller the scrolling animation effect is disabled when intercepting the event.

Will test and confirm.